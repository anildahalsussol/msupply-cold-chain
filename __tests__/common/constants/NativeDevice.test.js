import { DANGEROUS_BATTERY_LEVEL, BLUETOOTH_STATE, PERMISSION_STATE, PERMISSION } from '~constants';

describe('Constants: PERMISSION', () => {
  it('Matches snapshot', () => {
    expect(PERMISSION).toMatchSnapshot();
  });
});

describe('Constants: PERMISSION_STATE', () => {
  it('Matches snapshot', () => {
    expect(PERMISSION_STATE).toMatchSnapshot();
  });
});

describe('Constants: BLUETOOTH_STATE', () => {
  it('Matches snapshot', () => {
    expect(BLUETOOTH_STATE).toMatchSnapshot();
  });
});

describe('Constants: DANGEROUS_BATTERY_LEVEL', () => {
  it('Matches snapshot', () => {
    expect(DANGEROUS_BATTERY_LEVEL).toMatchSnapshot();
  });
});
