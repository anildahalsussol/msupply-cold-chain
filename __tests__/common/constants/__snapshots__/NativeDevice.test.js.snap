// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: BLUETOOTH_STATE Matches snapshot 1`] = `
Object {
  "OFF": "PoweredOff",
  "ON": "PoweredOn",
  "UNAUTHORIZED": "Unauthorized",
  "UNKNOWN": "Unknown",
  "UNSUPPORTED": "Unsupported",
}
`;

exports[`Constants: DANGEROUS_BATTERY_LEVEL Matches snapshot 1`] = `20`;

exports[`Constants: PERMISSION Matches snapshot 1`] = `
Object {
  "LOCATION": "android.permission.ACCESS_COARSE_LOCATION",
  "STORAGE": "android.permission.WRITE_EXTERNAL_STORAGE",
}
`;

exports[`Constants: PERMISSION_STATE Matches snapshot 1`] = `
Object {
  "BLOCKED": "blocked",
  "DENIED": "denied",
  "GRANTED": "granted",
  "UNAVAILABLE": "unavailable",
}
`;
